client/src/components/infoDrawer/infoFormat.js

```diff
@@ -43,11 +44,7 @@ const renderAffiliations = (affiliations, skeleton) => {
       <H3 className={skeleton ? Classes.SKELETON : null}>Affiliations</H3>
       <UL>
         {affiliations.map((item, index) => (
-          <div
-            id={`#afil${index}`}
-            key={item}
-            className={skeleton ? Classes.SKELETON : null}
-          >
+          <div key={item} className={skeleton ? Classes.SKELETON : null}>
```

Comment:
Is the `id` not necessary here?

Reply:
The id was left over from a previous attempt to reference it from the contributor list